docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -84,7 +84,33 @@ <h2>Rendering the scene</h2>
 
 		<h2>Animating the cube</h2>
 
-		<p>If you insert all the code above into the file you created before we began, you should see a green box. Let's make it all a little more interesting by rotating it.</p>
+		<p>If you insert all the code above into the file you created before we began the main.js file should look like</p>
+		<code>
+			import * as THREE from 'three';
+
+
+			const scene = new THREE.Scene();
+			const camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
+
+			const renderer = new THREE.WebGLRenderer();
+			renderer.setSize( window.innerWidth, window.innerHeight );
+			document.body.appendChild( renderer.domElement );
+
+
+			const geometry = new THREE.BoxGeometry( 1, 1, 1 );
+			const material = new THREE.MeshBasicMaterial( { color: 0x00ff00 } );
+			const cube = new THREE.Mesh( geometry, material );
+			scene.add( cube );
+
+			camera.position.z = 5;
+
+			function animate() {
+
+					renderer.render( scene, camera );
+			}
+			renderer.setAnimationLoop( animate );
+		</code>
+		<p>If your main.js snippet looks like the above then, you should see a green square. Let's make it all a little more interesting by rotating it.</p>
```

Comment:
TBH, I don't think the additional code snippet makes the guide more user friendly. It actually blows up the code listing so the focus is not on the actual important bit (the cube's animation). 

Besides, the complete example code comes a few lines below.